dev/prepare_bulk_issues.py

```diff
@@ -204,16 +205,12 @@ def prepare_bulk_issues(
     total_issues = len(names)
     processed_issues = 0
     if dry_run:
-        for name in names:
+        for name in itertools.islice(names, max_issues):
```

Comment:
```suggestion
        for name in names[:max_issues]:
```

`names` is already a list so a simple slicing is better